Header Header
string nodeName
uint8 desiredRate
tod_safety_monitoring/SafetyIssue[] issues